src/mainviews/detail-view/job-definition.tsx

```diff
@@ -33,22 +40,28 @@ export interface IJobDefinitionProps {
 
 export function JobDefinition(props: IJobDefinitionProps): JSX.Element {
   const trans = useTranslator('jupyterlab');
+  const [displayError, setDisplayError] = useState<string | undefined>();
```

Comment:
Same as in https://github.com/jupyter-server/jupyter-scheduler/pull/286/files#r1017133144